eox_tenant/filters/test/test_pipeline.py

```diff
@@ -161,3 +161,87 @@ def test_filter_render_certificates_by_org(self, organizations, render, mock_get
         else:
             FilterRenderCertificatesByOrg.run_filter(self, context, {})
             mock_get_organizations.assert_called_once()
+
+
+class FilterOrgAwareLMSURLStudioTestCase(TestCase):
+    """
+    FilterOrgAwareLMSURLStudioTestCase test cases.
+    """
+
+    def setUp(self):
+        """This method creates Microsite objects in database"""
+
+        # Creating mock to render tenant aware links
+        self.url = "https://lms-base"
+        self.org = "test"
+
+    @override_settings(
+        OPEN_EDX_FILTERS_CONFIG={
+            "org.openedx.learning.tenant_aware_link.render.started.v1": {
+                "fail_silently": False,
+                "pipeline": [
+                    "eox_tenant.filters.pipeline.OrgAwareLMSURLStudio"
+                ]
+            }
+        },
+        LMS_ROOT_URL="https://lms-base"
+    )
+    @mock.patch('eox_tenant.filters.pipeline.configuration_helpers')
+    def test_org_aware_lms_url_studio(self, configuration_helpers_mock):
```

Comment:
What do you think about using a more explicit name like: `test_get_lms_url_based_for_org`. Also, can we consider the case when the filter is not configured?

Reply:
Consider these comments for the other test suite as well.

Reply:
Thanks for removing it then.